packages/maskbook/src/plugins/Ideamarket/UI/NotListed.tsx

```diff
@@ -45,9 +46,8 @@ const useStyles = makeStyles((theme) => ({
 export default function NotListed(props: UIProps) {
     const classes = useStyles()
 
-    function clicked(e: MouseEvent) {
-        e.preventDefault()
-        window.open('https://app.ideamarket.io/', '_blank', 'noopener')
+    function clicked(e: SyntheticEvent) {
+        e.stopPropagation()
     }
```

Comment:
Actually, this part is also useless now.

Reply:
Same here. The tweet is getting clicked when I click on the button without this.

Reply:
Oh interesting. So please leave a comment so future devs know it is useful code.